### ISSUE TITLE
Quick Menu: Help file opened in Firefox and not web browser from "Preferred Applications"

### ISSUE THREAD

**Issue by sphh** [Odyseus/CinnamonTools#82]
### Description

The help file is not opened in the application set in "Preferred Application", but in Firefox.

### Steps to reproduce

1. Set web application in "Preferred Application" to something else than Firefox (I use Waterfox)
2. Right-click on Quick Menu icon and select Help.

**Expected behavior:**

I would expect that the help file opens in Waterfox (in my case).

**Actual behavior:**

The help file opens in Firefox.

**Reproduces how often:**

100%

### System information

inxi -xxxSc0 -! 31
```
System:    Kernel: 4.10.0-20-generic x86_64 (64 bit gcc: 5.4.0)
           Desktop: Cinnamon 3.2.7 (Gtk 3.18.9-1ubuntu3.3) dm: mdm
           Distro: Linux Mint 18.1 Serena
```

### System

The links `/etc/alternatives/x-www-browser` and `/etc/alternatives/gnome-www-browser` points to `/usr/bin/firefox`. I always thought that "Preferred Applications" works with the alternatives-system...

**Comment by Odyseus** [comment]
Hello, @sphh.

> I always thought that "Preferred Applications" works with the alternatives-system..

In my experience, the *"alternatives-system"* is more annoying than useful. You will have to set the *x-www-browser* "alternative" manually to Firefox with the following command:

```shell
sudo update-alternatives --config x-www-browser
```

Choose the number corresponding to Firefox and press <kbd>Enter</kbd>.

In my case, the *x-www-browser* "alternative" was set to auto. And every time that I installed a new browser, the default *x-www-browser* "alternative" changed to that last installed browser. The same happened to me with the *x-terminal-emulator* "alternative".

In short, the *"alternatives-system"* overrides the applications that we set as default on the system settings. And the systems settings doesn't change those on the *"alternatives-system"*. Like I said, annoying.

**Comment by sphh** [comment]
Thank you for your explanation. That's what I thought is behind this behaviour. Would you say that it is actually a bug with the "Preferred Application" / _alternatives system_ of Cinnamon?

**Comment by Odyseus** [comment]
Hello, @sphh.

I wouldn't say that it's a Cinnamon bug. Because, when I first found this annoying behavior, all that I could find *on the net* were related to Ubuntu or Debian and with different desktop environments.
